kairon/shared/channels/whatsapp/bsp/dialog360.py

```
@@ -281,4 +283,107 @@ async def upload_media(bot: str, bsp_type: str, media_id: str) -> str:
+            def _do():
+                with open(file_path, "rb") as f:
+                    files = {"file": (filename, f, f"{extension}")}
+
+                    return requests.post(
+                            f"{base_url}/media",
+                            headers=headers,
+                            data = payload,
+                            files = files,
+                            timeout = (5, 60),
+                            )
+            loop = asyncio.get_running_loop()
```

Ruff (0.12.2)

Undefined name `asyncio`

(F821)